pyproject.toml

```diff
@@ -28,12 +32,20 @@ dependencies = [
 ]
 
 [project.urls]
+homepage = "https://ramalama.ai"
 Repository = "https://github.com/containers/ramalama-stack"
 Issues = "https://github.com/containers/ramalama-stack/issues"
 
+[tool.setuptools_scm]
+version_file = "src/ramalama_stack/_version.py"
+# do not include +gREV local version, required for Test PyPI upload
+local_scheme = "no-local-version"
+
+[tool.setuptools]
+package-dir = { "" = "src" }
+
+[tool.setuptools.dynamic]
+dependencies = { file = ["requirements.txt"] }
```

Comment:
but the deps are in here too 🤔

Reply:
So maybe the requirements file should be removed altogether?

Reply:
Going to keep it for now as we try this with Test PyPI, can remove it later if needed